x/staking/keeper/keeper.go

```diff
@@ -485,6 +487,7 @@ func (k *Keeper) Unjail(ctx sdk.Context, valID hmTypes.ValidatorID) {
 	validator.Jailed = false
 
 	// add updated validator to store with new key
-	k.AddValidator(ctx, validator)
-	return
+	if err := k.AddValidator(ctx, validator); err != nil {
+		k.Logger(ctx).Error("Error calling AddValidator")
```

Comment:
```suggestion
		k.Logger(ctx).Error("Error calling AddValidator")
		return err
```

Reply:
Unjail method doesn't have any return types, leaving this as it was earlier.